examples/workflows/aws_privatelink/main.tf

```diff
@@ -36,7 +40,7 @@ variable "storage_gib" {
 variable "machine_type" {
   type     = string
   nullable = false
-  default  = "n1-standard-2"
+  default  = "m5.large"
```

Comment:
is this the smallest we can suggest?  Running this will cost someone real money so we should try to be sensitive to that.  I noticed this banner in the terraform sensitive variables example. It might be nice if we could add a disclaimer somewhere.
<img width="942" alt="Screen Shot 2022-11-03 at 3 48 58 PM" src="https://user-images.githubusercontent.com/6658984/199849693-ae82b8b1-9f30-4e06-8fbf-f46fd93b5f49.png">

Reply:
That's our default 2-vcpu cluster and the smallest of the standard options. All dedicated clusters cost money, and there is no dedicated free tier, but I guess we could add something to the readme.

Reply:
Added that disclaimer!

Reply:
great!